logprep/processor/geoip_enricher/rule.py

```diff
@@ -36,7 +37,61 @@ class Config(FieldManagerRule.Config):
         """RuleConfig for GeoipEnricher"""
 
         target_field: str = field(validator=validators.instance_of(str), default="geoip")
-        """Field for the output information. Defaults to :code:`geoip`"""
+        """Field for the output information. Defaults to :code:`geoip`."""
+        customize_target_subfields: dict = field(
+            validator=[
+                validators.instance_of(dict),
+                validators.deep_mapping(
+                    key_validator=validators.and_(
+                        validators.instance_of(str),
+                        validators.in_(
+                            [
+                                "type",
+                                "geometry.type",
+                                "geometry.coordinates",
+                                "properties.accuracy_radius",
+                                "properties.continent",
+                                "properties.continent_code",
+                                "properties.country",
+                                "properties.country_iso_code",
+                                "properties.time_zone",
+                                "properties.city",
+                                "properties.postal_code",
+                                "properties.subdivision",
```

Comment:
Well I could auto generate the documentation such that it is one location less to synchronize and I did revised it a bit, but I couldn't eliminate it completely. I can't add a constant into the `processor.py`, because I would then end up having circular imports. Please have a look at it again.